src/js/map/data-layers.js

```diff
@@ -481,7 +481,14 @@ class DataLayers {
     //for each layer in the config file
     for (const configLayer of this.mapConfig.layers) {
       //Get the right geoserver WFS link using the hostname
-      const url = this.mapClass.geoserver_wfs_url + configLayer.geoserverLayerName;
+      let url = '';
+      //If there is cql and it is not undefined, we add the cql filter to the wfs call
+      if (configLayer.cqlFilter && configLayer.cqlFilter !== 'undefined'){
```

Comment:
Could `if (configLayer.cqlFilter)` be enough? Just wondering in which scenario we would have a filter present in the json but being undefined.

Reply:
Change made